src/components/Clients/ClientList.jsx

```diff
@@ -35,7 +35,7 @@ const ClientList = () => {
         }}
       >
         <Typography variant="h6" component="h2" mb={2} align="center" color="secondary">
-          Add clients to your list
+        Add clients to your list
```

Comment:
Think we could add a space before "Add clients to your list" so that it'll be the same as before. That way it won't pop up as one of the files changed. Think running `npm run prettier:run` should do the trick not just for this file, but the rest of the files which might need formatting.